CMakeLists.txt

```diff
@@ -548,15 +550,16 @@ endif()
 
 # Add 'check' target - quick test
 if (NOT MFEM_USE_MPI)
-  add_custom_target(check
+  set(MFEM_CHECK_TARGET_NAME ${MFEM_CUSTOM_TARGET_PREFIX}check)
+  add_custom_target(${MFEM_CHECK_TARGET_NAME}
     ${CMAKE_CTEST_COMMAND} -R \"^ex1_ser\" -C ${CMAKE_CFG_INTDIR}
     USES_TERMINAL)
-  add_dependencies(check ex1)
+  add_dependencies(${MFEM_CHECK_TARGET_NAME} ex1)
 else()
-  add_custom_target(check
+  add_custom_target(${MFEM_CHECK_TARGET_NAME}
     ${CMAKE_CTEST_COMMAND} -R \"^ex1p\" -C ${CMAKE_CFG_INTDIR}
```

Comment:
@jandrej 
When MFEM_CHECK_TARGET_NAME is empty, CMAKE_TEST_COMMAND becomes the target name, and it's the path to ctest binary.
You need a default name for the check target.

Reply:
Line 553 handles this.  It wont be empty. it will either be just `check` or `${MFEM_CUSTOM_TARGET_PREFIX}check`

Reply:
Yeah, I was right and wrong at the same time :)